R/pull_NOMS_data.R

```diff
@@ -0,0 +1,45 @@
+library(googleAnalyticsR)
+library(googleAuthR)
+library(arrow)
+library(tidyr)
+library(lubridate)
+library(plyr)
+library(dplyr)
+library(urltools)
+
+get_nwis_site_page_views <- function(date_range = c('2020-01-01', '2020-02-01')) {
+  gar_auth_service('~/.vizlab/VIZLAB-a48f4107248c.json')
+  nwis_web_view <- '49785472'
+  
+  #filter 
+  #get all pagePaths with numeric data between 8 and 15 digits long in it:
+  filter_8_15_digit_site <- dim_filter("pagePath", "REGEXP", expressions = "[0-9]{8,15}")
```

Comment:
after removing the site_no part of the filter, you were right that the numeric filtering was redundant so I removed one of the filter clauses